cms/envs/common.py

```diff
@@ -1264,7 +1264,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
+HELP_TOKENS_INI_FILE = REPO_ROOT / "cms" / "envs" / "help_tokens.ini"
```

Comment:
Don't know if the 2 common.py files are the only place where the config.ini filenames are referenced? @nedbat?